docs/data/sql_functions_zh.yml

```diff
@@ -712,6 +712,12 @@ conversion:
       特别是匿名的内联数据类型没有可序列化的字符串表示。在这种情况下返回 `NULL`。
 
 collection:
+  - sql: ARRAY_SLICE(array, start_offset[, end_offset])
+    table: array.arraySlice(start_offset[, end_offset])
+    description: 返回输入数组的子数组，位于“start_offset”和“end_offset”之间
+      包含“start_offset”和“end_offset”）。偏移量从 1 开始，但 0 也被视为数组的开头。正值从数组开头开始计数，
+      负值从数组末尾开始计数。如果省略“end_offset”，则该偏移量将被视为数组的长度。如果“start_offset”在
+      “end_offset”之后或者两者都超出数组范围，则将返回空数组。如果任何输入为 null，则返回 null。
```

Comment:
nit: it's better to folow ABC order(by function name) like it's done in eglish version

Reply:
fix it.